.travis.yml

```diff
@@ -7,6 +7,94 @@ env:
 
 matrix:
   include:
+    - name: macOS 10.13, llvm 7
+      env: CC=/usr/local/opt/llvm@7/bin/clang
+      env: CXX=/usr/local/opt/llvm@7/bin/clang++
+      os: osx
+      osx_image: xcode10.1
+      sudo: false
+      addons:
+        homebrew:
+          packages:
+            - llvm@7
+            - libomp
+            - python
+          update: true
+
+    - name: macOS 10.13, GCC 8
+      env: CC=/usr/local/opt/gcc@8/bin/gcc-8
+      env: CXX=/usr/local/opt/gcc@8/bin/g++-8
+      os: osx
+      osx_image: xcode10.1
+      sudo: false
+      addons:
+        homebrew:
+          packages:
+            - gcc@8
+            - libomp
+            - python
+          update: true
+      before_install:
+        - brew link gcc@8
+
+    - name: macOS 10.13, Apple Clang
+      env: CC=cc
+      env: CXX=c++
+      os: osx
+      osx_image: xcode10.1
+      sudo: false
+      addons:
+        homebrew:
+          packages:
+            - libomp
+            - python
+          update: true
+
+    - name: macOS 10.12, llvm 7
+      env: CC=/usr/local/opt/llvm@7/bin/clang
+      env: CXX=/usr/local/opt/llvm@7/bin/clang++
+      os: osx
+      osx_image: xcode9.2
+      sudo: false
+      addons:
+        homebrew:
+          packages:
+            - llvm@7
+            - libomp
+            - python
+            - cmake
+          update: true
+
+    - name: macOS 10.12, GCC 8
+      env: CC=/usr/local/opt/gcc@8/bin/gcc-8
+      env: CXX=/usr/local/opt/gcc@8/bin/g++-8
+      os: osx
+      osx_image: xcode9.2
+      sudo: false
+      addons:
+        homebrew:
+          packages:
+            - gcc@8
+            - libomp
+            - python
+            - cmake
```

Comment:
in this build ```libomp``` as well as ```cmake```could be removed

Reply:
Addressed in 24e2c62e9f851be72a5e3bd4a19d8eb7d744e2a5